front/package.json

```
@@ -150,7 +150,7 @@
   },
   "scripts": {
     "compile": "tsc -b",
-    "start": "vite",
+    "start": "export NODE_OPTIONS='--max_old_space_size=4096' && vite",
```

nit: I would suggest switching to the additive form instead of overwriting NODE_OPTIONS, so that we preserve any existing flags a developer might have.
```suggestion
    "start": "NODE_OPTIONS=\"$NODE_OPTIONS --max-old-space-size=4096\" vite",
```